repo/repo.go

```diff
@@ -56,6 +56,10 @@ type Repo struct {
 
 	// Sum is the hash of the module to be verified after download.
 	Sum string
+
+	// Replace is the Go import path of the module configured by the replace
+	// directive in go.mod.
+	Replace string
```

Comment:
Please also add `replace` to `go_repository` `MergeableAttrs` in `language/go/kinds.go`. This will let Gazelle replace or remove existing `replace` attributes.

Reply:
all set